packages/elasticsearch-client/src/config/default/resources/index-document.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (C) 2021-2023 Technology Matters
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see https://www.gnu.org/licenses/.
+ */
+
+import { ReferrableResource } from '@tech-matters/types';
+
+export type ResourcesIndexDocument = {
+  name: string;
+  text1: string[];
+  text2: string[];
+};
+
+const HIGH_PRIORITY_ATTRIBUTES = ['title'];
+
+export const convertDocument = (resource: ReferrableResource): ResourcesIndexDocument => {
+  const { name } = resource;
+
+  const text1: string[] = [];
+
+  const text2: string[] = [];
+
+  const pushToCorrectText = (key: string, value: string) => {
+    if (HIGH_PRIORITY_ATTRIBUTES.includes(key)) {
+      text1.push(value);
+    } else {
+      text2.push(value);
+    }
+  };
+
+  const parseAttribute = (key: string, attribute: any) => {
```

Comment:
Is this function needed?

Reply:
No, it was leftover from some more complex earlier iterations but I ended up fighting with typescript for WAY too long when trying to remove it so I left it in for now. I added a TODO for the future "fix all of my lazy TS" iteration in a PR further down the chain.

If I'm honest, I still have no real idea what all of the possible `ResourceAttributeNode` types are.